src/test/txrebroadcast_tests.cpp

```diff
@@ -0,0 +1,193 @@
+// Copyright (c) 2011-2021 The Bitcoin Core developers
+// Distributed under the MIT software license, see the accompanying
+// file COPYING or http://www.opensource.org/licenses/mit-license.php.
+
+#include <amount.h>
+#include <boost/test/unit_test.hpp>
```

Comment:
Separate library imports from local project imports:

```suggestion
#include <boost/test/unit_test.hpp>

#include <amount.h>
```

Reply:
I separated them, but I have boost after internal. Is that ok? Btw, I don't see anything in the style guide about this, maybe we should add?

Reply:
Yes, that's the right way to do it.

I don't think we have any guidance in the style guide, but general best practice seems to be to go from small to large: https://stackoverflow.com/a/2762596/933705.